superset/charts/api.py

```diff
@@ -693,42 +692,3 @@ def thumbnail(
         return Response(
             FileWrapper(screenshot), mimetype="image/png", direct_passthrough=True
         )
```

Comment:
let's remove `datasources` from `include_route_methods` and `method_permission_name`